### ISSUE TITLE
Table cells become too wide when containing Br

### ISSUE THREAD

**Issue by iclukas** [speedata/publisher#194]
In table cells containing explicit line breaks (`Br` or HTML `br`), it seems the line width calculation is not reset at the breaks and thus gets too wide. It only works when the `Paragraph` is closed for every line.

![sp-table-br-bug](https://user-images.githubusercontent.com/1122393/44026830-aac64c06-9ef4-11e8-99c2-72e06cec95b4.png)

```
<Layout xmlns="urn:speedata.de:2009/publisher/en"
  xmlns:sd="urn:speedata:2009/publisher/functions/en">
  <Trace grid="no" objects="yes"/>

  <Record element="data">
    <PlaceObject>
      <Table padding="2mm" stretch="no" >
        <Tr>
          <Td>
            <Paragraph><Value>Row 1 / Column 1</Value><Br/>
            <Value>Lorem ipsum dolor sit amet, consectetur</Value><Br/>
            <Value>adipisicing elit, sed do eiusmod</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 1 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
        <Tr>
          <Td>
            <Paragraph><Value>Row 2 / Column 1</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 2 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
      </Table>
    </PlaceObject>
    <PlaceObject>
      <Table padding="2mm" stretch="no" >
        <Tr>
          <Td>
            <Paragraph><Value>Row 1 / Column 1<br/>Lorem ipsum dolor sit amet, consectetur<br/>adipisicing elit, sed do eiusmod</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 1 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
        <Tr>
          <Td>
            <Paragraph><Value>Row 2 / Column 1</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 2 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
      </Table>
    </PlaceObject>
    <PlaceObject>
      <Table padding="2mm" stretch="no" >
        <Tr>
          <Td>
            <Paragraph><Value>Row 1 / Column 1
Lorem ipsum dolor sit amet, consectetur
adipisicing elit, sed do eiusmod</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 1 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
        <Tr>
          <Td>
            <Paragraph><Value>Row 2 / Column 1</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 2 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
      </Table>
    </PlaceObject>
    <PlaceObject>
      <Table padding="2mm" stretch="no" >
        <Tr>
          <Td>
            <Paragraph><Value>Row 1 / Column 1</Value></Paragraph>
            <Paragraph><Value>Lorem ipsum dolor sit amet, consectetur</Value></Paragraph>
            <Paragraph><Value>adipisicing elit, sed do eiusmod</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 1 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
        <Tr>
          <Td>
            <Paragraph><Value>Row 2 / Column 1</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>Row 2 / Column 2</Value></Paragraph>
          </Td>
        </Tr>
      </Table>
    </PlaceObject>
  </Record>
</Layout>
```

**Comment by pgundlach** [comment]
This tries to be a minimal test case:


```xml
<Layout xmlns="urn:speedata.de:2009/publisher/en"
  xmlns:sd="urn:speedata:2009/publisher/functions/en">
  <Trace grid="no" objects="yes"/>

  <Record element="data">
    <PlaceObject>
      <Table>
        <Tr>
          <Td>
            <Paragraph><Value>text</Value></Paragraph>
          </Td>
          <Td>
            <Paragraph><Value>text<br/>text<br/>text</Value></Paragraph>
          </Td>
        </Tr>
      </Table>
    </PlaceObject>
  </Record>
</Layout>
```

<img width="118" alt="texttexttext" src="https://user-images.githubusercontent.com/209434/44029764-bf8819f8-9efe-11e8-8079-d2d2fd6382ac.png">

**Comment by pgundlach** [comment]
Fixed in version 3.3.12 which is online now
